examples/README.md

```diff
@@ -23,32 +23,44 @@ These examples are written in C#, but any [.NET language](https://en.wikipedia.o
 ## NAT example
 
 ### Configuration
-The configuration needs to specify two parameters to the NAT's port 0 (i.e., the port connected to "Outside"):
+The configuration needs to specify three parameters to the NAT's constructor:
 
 * The IP address of the NAT. This will be used as the source address of packets crossing from the Inside to the Outside of the NAT.
 * The starting TCP port to use on the NAT, to proxy TCP connections crossing from the Inside to the Outside of the NAT.
+* The MAC address of the next hop, the node which the Out port (port 0) of the NAT is connected to.
```

Comment:
Since there could be multiple next-hops (one for each port), I wonder if this point could be clarified by replace "next hop" with "next hop to Outside"?